src/main/java/com/spotinst/sdkjava/example/statefulnode/azure/AzureStatefulNodeExample.java

```diff
@@ -0,0 +1,558 @@
+
+        // Build Disk Deallocation Node
+        StatefulNodeDiskDeallocationConfig diskDeallocationConfig = StatefulNodeDiskDeallocationConfig.Builder.get().setShouldDeallocate(true).build();
+
+        // Build Snapshot Deallocation Node
+        StatefulNodeSnapshotDeallocationConfig snapchatDeallocationConfig = StatefulNodeSnapshotDeallocationConfig.Builder.get().setShouldDeallocate(true).build();
+
+        // Build PublicIp Deallocation Node
+        StatefulNodePublicIpDeallocationConfig publicIpDeallocationConfig = StatefulNodePublicIpDeallocationConfig.Builder.get().setShouldDeallocate(true).build();
+
+        StatefulNodeDeallocationConfig statefulDeallocationConfig = statefulDeallocationConfigBuilder.setShouldTerminateVms(true)
+                                       .setNetworkDeallocationConfig(networkDeallocationConfig).setDiskDeallocationConfig(diskDeallocationConfig)
+                                       .setSnapshotDeallocationConfig(snapchatDeallocationConfig).setPublicIpDeallocationConfig(publicIpDeallocationConfig).build();
+
+        // Build node deletion request
+        StatefulNodeDeletionRequest.Builder nodeDeletionRequestBuilder = StatefulNodeDeletionRequest.Builder.get();
+        StatefulNodeDeletionRequest         deletionRequest            = nodeDeletionRequestBuilder.setDeallocationConfig(statefulDeallocationConfig).build();
+
+        System.out.println(String.format("Stateful Node deletion Request: " + deletionRequest.toJson()));
+        Boolean deleteNodeResponse = client.deleteNode(deletionRequest,nodeId);
+
+        System.out.println(String.format("Stateful Node %s is successfully Deleted", nodeId));
+        return deleteNodeResponse;
+    }
+
+    private static List<StatefulNode> getAllStatefulNode(SpotinstAzureStatefulNodeClient client) {
+
+        // Get All Stateful Nodes
+        List<StatefulNode> statefulNodesList = client.getAllNodes();
+
+        System.out.println("Stateful Nodes List: ");
+        for (int i=0; i< statefulNodesList.size(); i++){
+            System.out.println(String.format("Stateful Node Id: %s and Name: %s ",statefulNodesList.get(i).getId() , statefulNodesList.get(i).getName()));
+
+        }
+
+        return statefulNodesList;
+
+    }
+
+    private static Boolean recycleStatefulNode(SpotinstAzureStatefulNodeClient client, String nodeId){
+
+        // Recycle Stateful Nodes
+        StatefulNodeStateRequest.Builder recycleStatefulNodeBuilder = StatefulNodeStateRequest.Builder.get();
+        StatefulNodeStateRequest recycleStatefulNode = recycleStatefulNodeBuilder.setState(AzureStatefulNodeStateEnum.RECYCLE).build();
+
+        Boolean recycleStatefulNodeResponse = client.updateStatefulNodeState(recycleStatefulNode,nodeId);
+
+        System.out.println(String.format("Recycle of stateful Node %s is successful ", nodeId));
```

Comment:
This message is not correct. The recycle may be failed also based on response of recycleStatefulNodeResponse

Reply:
Added Check of if(recycleStatefulNodeResponse)